changelog.d/294.change.rst

```diff
@@ -0,0 +1 @@
+``@attr.ib().default`` and ``@attr.ib().validate`` decorators now return ``self`` instead of ``meth``
```

Comment:
It’s validator, not validate.  We’ll fix convert eventually to achieve consistency to avoid this exact confusion.